compiler/tools/fpp-check/test/component/ok.fpp

```diff
@@ -39,4 +39,21 @@ active component C {
 
   telemetry port tlmOut
 
+  state machine LedSm
```

Comment:
For stylistic consistency let's use SM instead of LedSm.

Reply:
Fixed as specified in previous comment